### ISSUE TITLE
Command query

### ISSUE THREAD

**Issue by Twink-Polar** [Zetier/lariat#25]
I'm pushing a apk to an android device and then trying to install it e.g "adb install some.apk" but don't seem to getting much success. 

My lariat cmd: "lariat --command "install /data/local/tmp/some.apk". What am I missing?

Using latest v0.1.1.

**Comment by Twink-Polar** [comment]
That's the badger, it got my apk installed but lariat reported back "adb failure running command".

Think the command took longer than 10 seconds and reached a timeout?

**Comment by Twink-Polar** [comment]
I think my issue stems from the adb-shell python library. Lariat uses the class AdbDevice and function 'shell'. One of the timeout values defaults to 10 seconds. Any ideas how that could be increased?

**Comment by adam-zetier** [comment]
You are correct that it is hitting a read timeout on larger files and the default set by `adb_shell` is 10s: https://github.com/JeffLIrion/adb_shell/blob/master/adb_shell/constants.py#L122

You can avoid the error by manually increasing that timeout in the `device.connect` call by adding the parameter `read_timeout_s=30` here:  https://github.com/Zetier/lariat/blob/v0.1.1/src/lariat/lariat.py#L336

I am hesitant to make that an arbitrary value and may implement some logic to determine a timeout based on the size of the file being pushed.

Additionally, I believe we might add an alias `--install` to install APKs via push-file and pm install in a coming release. `adb_shell` currently does not directly support adb install currently; however, maybe a future version will.

**Comment by andy-zetier** [comment]
@Twink-Polar this issue should be resolved in release [v0.2.0](https://github.com/Zetier/lariat/releases/tag/v0.2.0).

You'll need to add a new `adb_shell_default_read_timeout_s` option to your lariat config file. For example:

```json
{
	"access_token": "5f120151fa6a43d3afaketoken2f659bb34ab1d9bab8104a",
	"device_farmer_url": "https://my.devicefarmer.com/",
	"adb_shell_default_read_timeout_s": 32.5
}
```

**Comment by Twink-Polar** [comment]
Thank you for adding this feature
